### DISCUSSION TITLE
How to get a Github API key?

### DISCUSSION THREAD

**Discussion by bioluminesceme** [OpenBB-finance/OpenBB#2062] [Q&A]
I've successfully entered AlphaVantage and Polygon but can't figure out how to generate an API key for Github, so that 'update' works.
Should it be a New personal access token via https://github.com/settings/tokens/new ? 
And what should the scope be? 

I've tried various scope settings, but OpenBB keeps saying "defined" and not that the test passed.

When I try update it indeed still says "Git dependencies not installed".

**Comment by DidierRLopes** [comment]
Hey @benninkcorien ,

The `update` command basically just does `git pull` from the source code directly.

You don't need to update the GITHUB token, but needs to have `git` configured. If you can do a `git clone ...` that means that update should be working.

Sorry for delay on this.

Hope this helps!

**Reply by bioluminesceme** [reply]
Thanks! I asked because I was trying to fix this error message, and the docs said something about a Github API token?
I've got git clone/pull/push etc working in my regular command prompt just fine.


Help showed

    update             update terminal automatically (when using GitHub)

But 

    2022 Jul 23, 09:32 (🦋) / $ update

    This feature is not available : Git dependencies not installed.
    resetting...
    usage: terminal

OpenBB Terminal v1.5.0m on Win11
